### PR TITLE
Fix ToString method for SvgArcSegment

### PR DESCRIPTION
**Pull request by wieslawsoltes**
#### What does this implement/fix? Explain your changes.

The `ToString` method used current culture which produces invalid path data on some culture (uses `,` instead of `.` for separator.

### CODE REVIEW

File: Source/Paths/SvgArcSegment.cs [hunk 1]
```diff
@@ -135,7 +136,7 @@ public override string ToString()
         {
             var arcFlag = Size == SvgArcSize.Large ? "1" : "0";
             var sweepFlag = Sweep == SvgArcSweep.Positive ? "1" : "0";
-            return "A" + RadiusX.ToString() + " " + RadiusY.ToString() + " " + Angle.ToString() + " " + arcFlag + " " + sweepFlag + " " + End.ToSvgString();
+            return "A" + RadiusX.ToString(CultureInfo.InvariantCulture) + " " + RadiusY.ToString(CultureInfo.InvariantCulture) + " " + Angle.ToString(CultureInfo.InvariantCulture) + " " + arcFlag + " " + sweepFlag + " " + End.ToSvgString();
```
Comment (mrbean-bremen) [inline Source/Paths/SvgArcSegment.cs]: Thanks! While you are it - could you check for more occurencies and fix them, too? I did a quick check, and found at least a couple more in `SvgOrient.ToString` and in `SvgOrientConverter.ConvertFrom`.

Comment (wieslawsoltes) [reply Source/Paths/SvgArcSegment.cs]: Fixed SvgOrient.ToString, not sure if SvgOrientConverter.ConvertFrom needs that.

Comment (wieslawsoltes) [reply Source/Paths/SvgArcSegment.cs]: I have checked other ToString methods and could not find any.

Comment (mrbean-bremen) [reply Source/Paths/SvgArcSegment.cs]: > not sure if SvgOrientConverter.ConvertFrom needs that.

I meant this line:
```C#
      if (!float.TryParse(value.ToString(), out fTmp))
```
As far as I know, this uses the current locale if not given one.

Comment (mrbean-bremen) [reply Source/Paths/SvgArcSegment.cs]: But maybe I'm wrong here... for some reason the culture info is given as an argument in `ConvertFrom` here, but not used.

Comment (wieslawsoltes) [reply Source/Paths/SvgArcSegment.cs]: Maybe we should leave this for now, as I am not convinced this needs invariant culture.

Comment (mrbean-bremen) [reply Source/Paths/SvgArcSegment.cs]: Agreed. Thanks for the help!

**Pull request merged**